components/form/BaseListbox.vue

```diff
@@ -255,111 +282,113 @@ const value = computed(() => {
 
         <div class="nui-listbox-outer">
           <FloatReference>
-            <ListboxButton
-              :disabled="props.disabled"
-              class="nui-listbox-button"
-            >
-              <slot name="listbox-button" :value="value" :open="open">
-                <div class="nui-listbox-button-inner">
-                  <BaseIconBox
-                    v-if="props.icon"
-                    size="xs"
-                    shape="rounded"
-                    class="nui-icon-box"
-                  >
-                    <slot name="icon">
-                      <Icon :name="props.icon" class="nui-icon-box-inner" />
-                    </slot>
-                  </BaseIconBox>
-
-                  <template v-if="Array.isArray(value)">
-                    <div
-                      v-if="value.length === 0 && placeholder"
-                      class="nui-listbox-placeholder"
-                      :class="props.loading && 'text-transparent select-none'"
-                    >
-                      {{ placeholder }}
-                    </div>
-                    <div
-                      class="block truncate text-left"
-                      :class="[
-                        props.loading && 'select-none text-transparent',
-                        value.length === 0 &&
-                          'text-muted-300 dark:text-muted-500',
-                      ]"
-                    >
-                      {{
-                        typeof props.multipleLabel === 'function'
-                          ? props.multipleLabel(value, props.properties.label)
-                          : props.multipleLabel
-                      }}
-                    </div>
-                  </template>
-
-                  <template v-else-if="value">
-                    <BaseAvatar
-                      v-if="
-                        props.properties.media &&
-                        (value as any)[props.properties.media]
-                      "
-                      :src="(value as any)[props.properties.media]"
-                      size="xs"
-                      class="-ms-2 me-2"
-                    />
+            <div>
```

Comment:
FYI, this div is necessary for the Float's adaptive width to capture the correct width of all of the elements